azure module go compilation error

Trying to run `go get -v -u github.com/gruntwork-io/terratest/modules/azure` fails. The error is below. This issue started after the release last night. I think `0.34.2` introduced the bug, but it could have been `0.34.1`. I am not confident.

```
/go/src/github.com/gruntwork-io/terratest/modules/azure/monitor.go:59:9: cannot use &settings (type *"github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2019-11-01-preview/insights".DiagnosticSettingsResource) as type *"github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2019-06-01/insights".DiagnosticSettingsResource in return argument
/go/src/github.com/gruntwork-io/terratest/modules/azure/monitor.go:103:9: cannot use &status (type *"github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2019-11-01-preview/insights".VMInsightsOnboardingStatus) as type *"github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2019-06-01/insights".VMInsightsOnboardingStatus in return argument
/go/src/github.com/gruntwork-io/terratest/modules/azure/monitor.go:156:9: cannot use &activityLogAlertResource (type *"github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2019-11-01-preview/insights".ActivityLogAlertResource) as type *"github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2019-06-01/insights".ActivityLogAlertResource in return argument
```

--- a/modules/azure/client_factory.go
+++ b/modules/azure/client_factory.go
@@ -15,10 +15,10 @@ import (
 	"reflect"
 
 	"github.com/Azure/azure-sdk-for-go/profiles/preview/cosmos-db/mgmt/documentdb"
-	"github.com/Azure/azure-sdk-for-go/profiles/preview/preview/monitor/mgmt/insights"
 	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-07-01/compute"
 	"github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2019-11-01/containerservice"
 	kvmng "github.com/Azure/azure-sdk-for-go/services/keyvault/mgmt/2016-10-01/keyvault"
+	"github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2019-03-01/insights"
 	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-06-01/subscriptions"
 	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2019-06-01/storage"
 	autorestAzure "github.com/Azure/go-autorest/autorest/azure"

--- a/modules/azure/monitor.go
+++ b/modules/azure/monitor.go
@@ -3,7 +3,7 @@ package azure
 import (
 	"context"
 
-	"github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2019-06-01/insights"
+	"github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2019-03-01/insights"
 	"github.com/gruntwork-io/terratest/modules/testing"
 	"github.com/stretchr/testify/require"
 )

